Util_UsageStatistics.php

```diff
@@ -1,218 +1,402 @@
-			if ( !empty( $item_value ) ) {
+			if ( ! empty( $item_value ) ) {
 				$v += $item_value;
 				$count++;
 			}
 		}
 		return ( $count <= 0 ? 0 : $v / $count );
 	}
 
-
-
 	/**
-	 * Sum up all positive metric values which names start with specified prefix
-	 **/
-	static public function sum_by_prefix_positive( &$output, $history, $property_prefix ) {
+	 * Sums up all positive values of properties whose names start with a specific prefix.
+	 *
+	 * This method iterates through an array of arrays and aggregates values of keys that start
+	 * with the given prefix. Only positive values are included in the sum.
+	 *
+	 * @param array  $output          The array to store the aggregated sums by property name.
+	 * @param array  $history         An array of arrays to sum values from.
+	 * @param string $property_prefix The prefix of the property names to match.
+	 *
+	 * @return void
+	 */
+	public static function sum_by_prefix_positive( &$output, $history, $property_prefix ) {
 		$property_prefix_len = strlen( $property_prefix );
 
 		foreach ( $history as $i ) {
 			foreach ( $i as $key => $value ) {
-				if ( substr( $key, 0, $property_prefix_len ) == $property_prefix &&
-					$value > 0 ) {
-					if ( !isset( $output[$key] ) ) {
-						$output[$key] = 0;
+				if ( substr( $key, 0, $property_prefix_len ) === $property_prefix && $value > 0 ) {
+					if ( ! isset( $output[ $key ] ) ) {
+						$output[ $key ] = 0;
 					}
 
-					$output[$key] += $value;
+					$output[ $key ] += $value;
 				}
 			}
 		}
 	}
 
-
-
-	static public function time_mins( $timestamp ) {
-		return date( 'm/d/Y H:i', $timestamp );
+	/**
+	 * Formats a timestamp into a human-readable date and time.
+	 *
+	 * This method uses PHP's `date()` function to format the provided timestamp in
+	 * the 'm/d/Y H:i' format.
+	 *
+	 * @param int $timestamp The Unix timestamp to format.
+	 *
+	 * @return string The formatted date and time string.
+	 */
+	public static function time_mins( $timestamp ) {
+		return gmdate( 'm/d/Y H:i', $timestamp );
 	}
 
-
-
-	static public function integer( $v ) {
+	/**
+	 * Formats an integer value with thousand separators.
+	 *
+	 * This method uses `number_format()` to add separators to large integers for readability.
+	 *
+	 * @param int|float $v The value to format.
+	 *
+	 * @return string The formatted number as a string.
+	 */
+	public static function integer( $v ) {
 		return number_format( $v );
 	}
 
-
-
-	static public function integer_divideby( $v, $divide_by ) {
-		if ( $divide_by == 0 ) {
+	/**
+	 * Divides a value by a specified divisor and formats the result as an integer with thousand separators.
+	 *
+	 * If the divisor is zero, the method returns 'n/a'.
+	 *
+	 * @param float|int $v         The value to divide.
+	 * @param float|int $divide_by The divisor.
+	 *
+	 * @return string The formatted result, or 'n/a' if the divisor is zero.
+	 */
+	public static function integer_divideby( $v, $divide_by ) {
+		if ( 0 === $divide_by ) {
```

Comment:
`$divide_by` could be an `int` or `float` -- change to `if ( empty( $divide_by ) ) {`.

Reply:
Updated